charts/kube-master/templates/api.yaml

```diff
@@ -356,6 +358,13 @@ spec:
             - mountPath: /etc/kubernetes/config
               name: wormhole-config
               readOnly: true
+          livenessProbe:
+            exec:
+              command:
+              - cmp -s /etc/kubernetes/certs/tls-wormhole.pem /tmp/tls-wormhole.livenessprobe
```

Comment:
you are sure this works?
I think this should be:
```
- cmp
- -s
- /etc/kubernetes/certs/tls-wormhole.pem
- /tmp/tls-wormhole.livenessprobe
```

or `[cmp, -s, /etc/kubernetes/certs/tls-wormhole.pem, /tmp/tls-wormhole.livenessprobe]`

Reply:
sigh, of course you are right (facepalm)